services/121-service/seed-data/program/program-test.json

```diff
@@ -211,5 +211,6 @@
   "aboutProgram": {
     "en": "[about aid program]"
   },
-  "fullnameNamingConvention": ["name"]
+  "fullnameNamingConvention": ["name"],
+  "languages": ["en"]
```

Comment:
I would suggest to have "at least more then 1" language available in the test-setup.